Lib/pathlib.py

```diff
@@ -233,14 +233,10 @@ def __repr__(self):
         return "<{}.parents>".format(type(self._path).__name__)
 
 
-class PurePath(object):
-    """Base class for manipulating paths without I/O.
+class _BasePurePath(object):
```

Comment:
If we're touching this line anyway...

```suggestion
class _BasePurePath:
```